js/client-mainmenu.js

```diff
@@ -896,8 +895,12 @@
 			var formatid = this.sourceEl.closest('form').find('button[name=format]').val();
 			i = +i;
 			this.sourceEl.val(i).html(TeamPopup.renderTeam(i));
-			app.rooms[''].curTeamIndex = i;
-			app.rooms[''].curTeamFormat = formatid;
+			if (this.sourceEl[0].offsetParent.className === 'mainmenuwrapper') {
+				app.rooms[''].curTeamIndex = i;
+				app.rooms[''].curTeamFormat = formatid;
+			} else if (this.sourceEl[0].offsetParent.className === 'tournament-box active') {
+				app.curSideRoom.tournamentBox.curTeamIndex = i;
```

Comment:
Tournaments don't have to be created in side rooms; your window might not be large enough for a side room, or you may have shifted the tab to the main side.